pycargoebuild/ebuild.py

```diff
@@ -219,8 +257,14 @@ def get_ebuild(pkg_meta: PackageMetadata,
     template = EBUILD_TEMPLATE_START
     if crate_license:
         template += EBUILD_TEMPLATE_CRATE_LICENSE
+    if features:
+        template += EBUILD_TEMPLATE_FEATURES
     template += EBUILD_TEMPLATE_END
 
+    iuse = get_IUSE(pkg_meta.features)
+    if pkg_meta.features and iuse and features:
```

Comment:
```suggestion
    if iuse and features:
```

`get_IUSE()` is already supposed to handle "no features" case, isn't it?